api/config/feature-toggles-config.js

```diff
@@ -35,4 +35,10 @@ export default {
     defaultValue: false,
     tags: ['frontend', 'pix-app'],
   },
+  isV3CertificationPageEnabled: {
+    description: 'Used to enable new certification page for V3',
+    type: 'boolean',
+    defaultValue: false,
+    tags: ['frontend', 'team-certification'],
```

Comment:
```suggestion
    tags: ['frontend', 'pix-app', 'team-certification'],
```

Question: Bien que non nécessaire, est-ce que l'on ajouterait pas le tag concernant l'application front impactée pour facilement le voir, comme cela a l'air d'être le cas pour les autres FT ?